### ISSUE TITLE
[welle-cli] Slides from SLS not shown?

### ISSUE THREAD

**Issue by andimik** [AlbrechtL/welle.io#443]
When I click on the red SLS button the slides are not shown.

![image](https://user-images.githubusercontent.com/24510556/67323457-c2ec6700-f512-11e9-818c-e5462b109a4a.png)

Tested with various DAB streams containing JPG and PNG files.

For example 
http://localhost:8081/slide/41949

seems to be 0 Bytes.

> The image “http://localhost:8081/slide/41949” cannot be displayed because it contains errors.

Tested with

```
$ welle-cli -f ~/stdin.raw -Dw 8081
Hello this is welle-cli v2.0-33-g5385057
```

**Comment by seife** [comment]
Fixed by pr #495 ;-)

**Comment by andimik** [comment]
Yes, confirmed!
